backend-lists.md

```diff
@@ -491,6 +493,50 @@ In the above example, the `city` scope will refresh when the `country` scope has
 
 > **Note:** Scope dependencies with `type: group` are only supported at this stage.
 
+<a name="custom-variables"></a>
+### Available custom variables for filters
+
+October CMS comes with some built in custom variables with each scope type.
+
+<div class="content-list collection-method-list" markdown="1">
+- [Date](#custom-variables-date)
+- [Date range](#custom-variables-daterange)
+- [Number](#custom-variables-number)
+- [Number range](#custom-variables-numberrange)
+- [Text](#custom-variables-text)
+</div>
+
+<a name="custom-variables-date"></a>
+### date
+
+- `:filtered`
+- `:after`
+- `:before`
+
+<a name="custom-variables-daterange"></a>
+### daterange
+
+- `:afterDate`
+- `:after`
+- `:beforeDate`
+- `:before`
+
+<a name="custom-variables-number"></a>
+### number
+
+- `:filtered`
+
+<a name="custom-variables-numberrange"></a>
+### numberrange
+
+- `:min`
+- `:max`
+
+<a name="custom-variables-text"></a>
+### text
+
+- `:value`
+
```

Comment:
I think it would be better if the available variables to be used in the condition property were documented alongside each filter type instead of a separate section. Something like "the following values are available to be used in the condition property for this filter type"

Reply:
Done